### PR TITLE
SB-29494 - Configuration param to check index file in Zip content

### PR DESCRIPTION
**Pull request by karthik-tarento**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-29494" title="SB-29494" target="_blank">StorySB-29494</a>  Need configuration parameter for Index file check for zip file
  </td></table>
  <br />
 

PR for JIRA #SB-29494

### CODE REVIEW

**Comment by karthik-tarento** [comment]
@AmiableAnil - Please review this PR.. this is replacement for PR #818

File: content-api/content-service/conf/application.conf [hunk 1]
```diff
@@ -738,4 +738,7 @@ collection {
 }
 
 #Request timeout
-actor.timeoutMillisec = 30000 
\ No newline at end of file
+actor.timeoutMillisec = 30000 
+
+#Index file validation
+isIndexHtmlValidationRequired=false
```
Comment (AmiableAnil) [inline content-api/content-service/conf/application.conf:744]: Initialize with true by default.

**Comment by AmiableAnil** [comment]
@karthik-tarento Also verify the review and publish flow when **isIndexHtmlValidationRequired** is false.

**Comment by karthik-tarento** [comment]
> @karthik-tarento Also verify the review and publish flow when **isIndexHtmlValidationRequired** is false.

Yes, this is verified in our environment.

**Review by AmiableAnil — APPROVED** [review]

**Pull request merged**